sites/svelte.dev/src/routes/auth/callback.js

```diff
@@ -44,7 +44,7 @@ export async function get({ host, query }) {
 					expires: new Date(expires),
 					path: '/',
 					httpOnly: true,
-					secure: !host.startsWith('localhost:')
+					secure: !url.host.startsWith('localhost:')
```

Comment:
Would it make sense to use `url.protocol` here (and in the other places we're checking for `localhost`)?

Reply:
Looks like `url.protocol` is "http:", but `url.hostname` is "localhost" (for http://localhost:3000). Did you mean the latter?

Reply:
I think he means to change the check to `secure: url.host.startsWith('https:')`, which sounds like a good idea to me

Reply:
Ah, gotcha. Will update.

Reply:
See https://github.com/sveltejs/sites/pull/270/commits/9f8a47f6b0201d29fe03d44138660eae507f4524 -- lmk if that's not what you were looking for.